js/components/typeahead.js

```diff
@@ -0,0 +1,160 @@
+Fliplet.FormBuilder.field('typeahead', {
+  name: 'Typeahead (multi-select)',
+  category: 'Multiple options',
+  props: {
+    value: {
+      type: Array
+    },
+    defaultValue: {
+      type: String,
+      default: ''
+    },
+    description: {
+      type: String
+    },
+    placeholder: {
+      type: String
+    },
+    optionsType: {
+      type: String,
+      default: 'dataSource'
+    },
+    maxItems: {
+      type: Number,
+      default: null
+    },
+    options: {
+      type: Array,
+      default: function() {
+        return [
+          {
+            id: '1',
```

Comment:
Why is this given a different ID? I'm comparing this with other multi-option fields like https://github.com/Fliplet/fliplet-widget-form-builder/blob/27f5e2bc6810a126e490d06515807c8bdf66c79b/js/components/checkbox.js#L21